service/framework.go

```diff
@@ -251,6 +251,7 @@ func newCustomObjectFramework(config Config) (*framework.Framework, error) {
 	versionedResources := map[string][]framework.Resource{
 		key.LegacyVersion:         legacyResources,
 		key.CloudFormationVersion: resources,
+		"1.0.0":                   resources,
```

Comment:
This should be `legacyResources`. The cloud formation changes aren't ready to expose to customers yet.

Reply:
Ack.